### DISCUSSION TITLE
Default value is not working as expected

### DISCUSSION THREAD

**Discussion by eeco4poa** [sqlitecloud/discussions#40] [Bug Report]
In sqlite cloud I run "ALTER TABLE "FILE" add column NEWCOLUMN_1 char(5) default "15"". 
It creates the NEWCOLUMN with value 15 to the existed records but whwn I enter a new record NEWCOLUMN is blank.
Is it a bug ?

**Comment by marcobambini** [comment]
@eeco4poa, can you please provide us with more details? What do you mean "When I enter a new record?"

**Reply by eeco4poa** [reply]
Of course. 

This is the table Employee with "ID_ETOS" INTEGER DEFAULT 1.

CREATE TABLE "EMPLOYEE" (
	"ID_EMPLOYEE"	INTEGER,
	"FIRST_NAME"	TEXT,
	"LAST_NAME"	TEXT,
	"PHONE"	TEXT,
	"EMAIL"	TEXT,
	"ID_IDIOTHTA"	INTEGER,
	"ID_YPOURGOS"	INTEGER,
	"ID_YFYPOURGOS"	INTEGER,
	"ID_GENERAL_SECRETARIAT"	INTEGER,
	"ID_GENERAL_DIRECTORATE"	INTEGER,
	"ID_DIRECTORATE"	INTEGER,
	"ID_DEPARTMENT"	INTEGER,
	"ID_ETOS"	INTEGER DEFAULT 1,
	PRIMARY KEY("ID_EMPLOYEE"),
	FOREIGN KEY("ID_IDIOTHTA") REFERENCES "IDIOTHTA"("ID_IDIOTHTA")
);

And this is a prn scr (two parts) after entering and saving a new record (**using the + sign in the bottom of the screen**) :


![image](https://github.com/user-attachments/assets/37f24b32-859a-4e3a-afde-a026c092de1f)

![image](https://github.com/user-attachments/assets/b3ed2443-f9d6-4fb1-bb1e-075cf95c30e8)

As you can see ID_ETOS has no value 1.

On the contrary when I create a new variable with default value, all the records take this default value (but not the new entries).

Hope I helped
